src/Common/MemoryTracker.h

```diff
@@ -4,6 +4,7 @@
 #include <chrono>
 #include <base/types.h>
 #include <Common/CurrentMetrics.h>
+#include <Common/CurrentMemoryTracker.h>
```

Comment:
It is better to avoid this, since changing `CurrentMemoryTracker.h` will recompile 2K modules, while changing MemoryTracker.h will change 1K